### ISSUE TITLE
Plugins from "geany-plugins-1.36_setup.exe" are not visible

### ISSUE THREAD

**Issue by pkitszel** [geany/geany-plugins#967]
Setup:
Windows 10
fresh installation of geany and geany-plugins

After plugins installation I see additional folders like C:\Program Files (x86)\Geany\share\geany-plugins\git-changebar
but there are only 6 default plugins available in Tools -> Plugins menu.

I also tried with geany 1.35, and without any custom config (user/appdata/roaming/geany).

Maybe it is caused by my windows installation, but I'm not an owner of this machine.

Where should I look to solve this issue?

**Comment by eht16** [comment]
A few questions:
a) did you deselect the "Depedencies" choice in the Geany-Plugins installer? If so, this might be the reason.
b) do you have all plugin DLLs in C:\Program Files (x86)\Geany\lib\geany (more than the six bundled plugins?
c) if not a) but b) then could you paste the output of Help>Debug Messages? This should give a clue why available plugins cannot be loaded

**Comment by pkitszel** [comment]
Thank you for your interest.

Both a) and b) are set properly.
Below is debug messages: https://www.geany.org/p/QSKrw/

**Comment by eht16** [comment]
Hmm, your logs says `System plugin path: C:\Users\me\Geany 1.36\lib\geany`.
In the first post you said `C:\Program Files (x86)\Geany\share\geany-plugins\git-changebar`, these paths don't match.
Did you change the installation prefix in the meantime or might this be the problem?
Can you post the directory contents of `C:\Users\me\Geany 1.36\lib\geany` (or whereever Geany is installed).

Did you install Geany and Geany-Plugins into the same prefix? This is the recommended way.

**Comment by pkitszel** [comment]
Nice catch!
It turned out that I have two geany instances installed, but plugins were installed only for one of them.
Solved!

Full story:
I first installed geany "only for user", because I didn't have admin permissions at the moment. Later I got permissions, and finally installed plugins in default path.
All attempts at reinstallation was taken on Geany from Program Files, but shortcut was starting old, user-only instance.
